google-cloud-asset/samples/quickstart.rb

```diff
@@ -99,3 +99,53 @@ def create_feed project_id:, feed_id:, pubsub_topic:, asset_names:
   puts "Created feed: #{response.name}"
   # [END asset_quickstart_create_feed]
 end
+
+def search_all_resources scope: "", query: "", asset_types: [], page_size: 0, page_token: "", order_by: ""
+  # [START asset_quickstart_search_all_resources]
+  require "google/cloud/asset/v1"
+
+  # scope = 'SCOPE_OF_THE_QUERY'
+  # query = 'QUERY_STATEMENT'
+  # asset_types = 'AN_ARRAY_OF_ASSET_TYPES_TO_SEARCH_FOR'
+  # page_size = 'SIZE_OF_EACH_RESULT_PAGE'
+  # page_token = 'TOKEN_PRODUCED_BY_THE_PRECEDING_CALL'
+  # order_by = 'FIELDS_TO_SORT_THE RESULTS'
+  asset_service = ::Google::Cloud::Asset::V1::AssetService::Client.new
```

Comment:
Please use `Google::Cloud::Asset.asset_service` instead, following the other quickstart examples.